ruby_connectors/lib/ruby_connectors/temporal/activities/read_api_data_activity.rb

```
@@ -0,0 +1,48 @@
+          start_to_close: 1800, # 30 minutes in seconds
+          schedule_to_close: 2000,  # ~33 minutes
+          schedule_to_start: 120,   # 2 minutes
+          heartbeat: 120 # 2 minutes
+        )
+
+        def execute(params)
+          params = params.transform_keys(&:to_s)
+          workflow_store = Services::Redis::WorkflowStoreService.new
+
+          begin
+            connector_class = Object.const_get("RubyConnectors::#{params["connector_class_name"].capitalize}Connector::Reader")
```

rubocop (1.68.0)

[convention] Line is too long. [127/120]

(Layout/LineLength)